### DISCUSSION TITLE
Bad quality

### DISCUSSION THREAD

**Discussion by pablosalinasf** [TheLastBen/fast-stable-diffusion#589] [Q&A]
Lately, I've noticed that some days the quality of the images is much lower.
I tried the same seed, prompt and exactly the same settings of two images generated at different moments and got quite different results.
Is anybody having this issue too? Thank you!

![00008-836960280-midjourneyart style, portrait of a sexy woman, making love, lust, orgasm, surrounded by insects by karol bak, james jean, tom ba](https://user-images.githubusercontent.com/118823307/203324059-d15bf98f-fc0e-4ef2-a532-f6f3eb436061.png)
![00055-836960280-midjourneyart style, portrait of a sexy woman, making love, lust, orgasm, surrounded by insects by karol bak, james jean, tom ba](https://user-images.githubusercontent.com/118823307/203324065-820ae2b7-149a-4997-bc71-3a3c742eca6d.png)

**Comment by TheLastBen** [comment]
A1111 colab ?

**Reply by pablosalinasf** [reply]
Yes
